arcade/examples/sprite_health.py

```diff
@@ -1,230 +1,249 @@
-
-        # Variables that will hold sprite lists
-        self.player_list = None
-        self.coin_list = None
-        self.bullet_list = None
-
-        # Set up the player info
-        self.player_sprite = None
-        self.score = 0
-
-        # Don't show the mouse cursor
-        self.set_mouse_visible(False)
-
-        # Load sounds. Sounds from kenney.nl
-        self.gun_sound = arcade.load_sound(":resources:sounds/hurt5.wav")
-        self.hit_sound = arcade.load_sound(":resources:sounds/hit4.wav")
-        self.death_sound = arcade.load_sound(":resources:sounds/hit5.wav")
-
-        arcade.set_background_color(arcade.color.AMAZON)
-
-    def setup(self):
-
-        """ Set up the game and initialize the variables. """
-
-        # Sprite lists
-        self.player_list = arcade.SpriteList()
-        self.coin_list = arcade.SpriteList()
         self.bullet_list = arcade.SpriteList()
+        self.bar_list = arcade.SpriteList()
+        self.player_sprite = Player(":resources:images/animated_characters/female_person/femalePerson_idle.png", SPRITE_SCALING_PLAYER, self.bar_list)
+        self.enemy_sprite = arcade.Sprite(":resources:images/animated_characters/zombie/zombie_idle.png", SPRITE_SCALING_ENEMY)
+        self.enemy_timer = 0
 
-        # Set up the player
-        self.score = 0
-
-        # Image from kenney.nl
-        self.player_sprite = arcade.Sprite(":resources:images/animated_characters/female_person/femalePerson_idle.png",
-                                           SPRITE_SCALING_PLAYER)
-        self.player_sprite.center_x = 50
-        self.player_sprite.center_y = 70
-        self.player_list.append(self.player_sprite)
-
-        # Create the coins
-        for i in range(COIN_COUNT):
-
-            # Create the coin instance
-            # Coin image from kenney.nl
-            coin = SpriteWithHealth(":resources:images/items/coinGold.png",
-                                    SPRITE_SCALING_COIN,
-                                    max_health=5)
-
-            # Position the coin
-            coin.center_x = random.randrange(SCREEN_WIDTH)
-            coin.center_y = random.randrange(150, SCREEN_HEIGHT)
-
-            # Add the coin to the lists
-            self.coin_list.append(coin)
+    def setup(self) -> None:
+        """Set up the game and initialize the variables."""
+        # Setup player and enemy positions
+        self.player_sprite.position = self.width // 2, self.height // 4
+        self.enemy_sprite.position = self.width // 2, self.height // 2
 
         # Set the background color
-        arcade.set_background_color(arcade.color.AMAZON)
-
-    def on_draw(self):
-        """
-        Render the screen.
-        """
+        self.background_color = arcade.color.AMAZON
 
-        # This command has to happen before we start drawing
+    def on_draw(self) -> None:
+        """Render the screen."""
+        # Clear the screen. This command has to happen before we start drawing
```

Comment:
I would add a blank line between the docstring and the comment to increase legibility.